pkg/kubelet/sysctl/runtime.go

```diff
@@ -70,16 +72,22 @@ func NewRuntimeAdmitHandler(runtime container.Runtime) (*runtimeAdmitHandler, er
 				Message: "Docker before 1.12 does not support sysctls",
 			},
 		}, nil
+	case rktTypeName:
+		return &runtimeAdmitHandler{
+			result: lifecycle.PodAdmitResult{
+				Admit:   false,
+				Reason:  UnsupportedReason,
+				Message: "Rkt does not support sysctls",
+			},
+		}, nil
+	default:
+		// Return admit for other runtimes.
+		return &runtimeAdmitHandler{
+			result: lifecycle.PodAdmitResult{
```

Comment:
Which brings up the question will kube be adding incubator runtimes to kube's CI :-)